azerothcore/azerothcore-wotlk
# feat(DB/*_dbc): Update loc column names to reflect blizzlike assignments.

## Changes Proposed:
-  Changes ordering and names of localized column names in *_dbc tables.
Original order:
`enUS, enGB, koKR, frFR, deDE, enCN, zhCN, enTW, zhTW, esES, esMX, ruRU, ptPT, ptBR, itIT, Unk`
Proposed order:
`enUS/enGB, koKR, frFR, deDE, enCN/zhCN, zhTW, esES, esMX, ruRU, Unk, Unk, Unk, Unk, Unk, Unk, Unk`
Note that for the first and fifth column in the PR, only one locale is in the column name. E.g. `Name_Lang_zhCN`.

## Issues Addressed:

- Closes unreported issue.

## SOURCE:

DBCs from various localized clients.

## Tests Performed:

- Tested that the server would not crash on loading the unfilled tables.

## How to Test the Changes:

1. Run the query.
2. Run the worldserver and make sure it runs as you'd expect.
Filling the tables shouldn't be necessary, since I don't know of any situation where the localized fields get sent to the client or are used for anything visible to the client.

## Known Issues and TODO List:


- [ ] enTW locale is nowhere to be found. It's very likely it's a part of that sixth column, but not known for certain.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
